package-scripts.js

```diff
@@ -23,8 +24,25 @@ function test(testName, mochaParams) {
 module.exports = {
   scripts: {
     build: {
-      script: `browserify -e browser-entry.js --plugin ./scripts/dedefine --ignore './lib/cli/*.js' --ignore 'chokidar' --ignore 'fs' --ignore 'glob' --ignore 'path' --ignore 'supports-color' -o mocha.js`,
-      description: 'Build browser bundle'
+      default: {
+        script: 'nps build.tsc build.copy build.browserify',
+        description: 'Build'
+      },
+      tsc: {
+        script: 'tsc -p . || true',
```

Comment:
FYI, you don't need to compile with TS. You can set `noEmit: true` or `emitDeclarationsOnly: true` and avoid transpiling the JS. I imagine that's better for keeping the project simpler?

Reply:
Yeah, I wanted to give it a try just to see if everything still works, but you're right.  I'll make it clearer that we don't need to do this.